src/js/gi/components/profile/CautionaryInformation.jsx

```diff
@@ -1,43 +1,52 @@
 import React from 'react';
-import { connect } from 'react-redux';
-import { showModal } from '../../actions';
+
 import AlertBox from '../../../common/components/AlertBox';
 
 const TableRow = ({ description, thisCampus, allCampuses }) => {
+  if (!thisCampus && !allCampuses) return null;
+  const bold = description === 'Total Complaints';
   return (
     <tr>
-      <th><strong>{description}</strong></th>
-      <td className="number">{thisCampus}</td>
-      <td className="number">{allCampuses}</td>
+      <th><b>{description}</b></th>
```

Comment:
I chose `<b>` here since it's not really used for emphasis relative to its surrounding elements. I think of `<strong>` being used in a phrase or sentence.